lib/suspenders/generators/production/single_redirect.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "../base"
+
+module Suspenders
+  module Production
+    class SingleRedirect < Generators::Base
+      def add_rack_canonical_host
+        inject_into_file(
+          "config/environments/production.rb",
+          %{\n  config.middleware.use Rack::CanonicalHost, ENV.fetch("APPLICATION_HOST")},
```

Comment:
Did you intend for `ENV.fetch("APPLICATION_HOST")` to be evaluated? If so, I believe you need to update this to read: `#{ENV.fetch("APPLICATION_HOST")}`

Reply:
ah, disregard :) I see that the intent is to add `ENV.fetch("APPLICATION_HOST")` and not the retrieved value. Makes sense 👍